web-client/public/js/graph.js

```diff
@@ -112,46 +124,71 @@ var drawGraph = function (nodes, links, positiveWeights, negativeWeights, sheetT
                      .append("g")
 
 
+  var previousValue = zoom.translate().slice();
+  var previousMax = [width, height];
+  var previousScale = zoom.scale();
   function zoomed(manual = false) {
```

Comment:
I know this was not changed in this PR, but note that default parameters (i.e., `manual = false`) are an ES6-only feature. This has probably gone unnoticed because this is browser-side code and most browsers are now ES6-capable, but still worth noting because everywhere else, we are still at ES5.